lib/model/classycat_classify.py

```diff
@@ -233,19 +273,23 @@ def process(self, message: Message) -> ClassyCatBatchClassificationResponse:
             else:
                 raise PrestoBaseException(f"Error classifying items: {e}", 500) from e
 
-
     @classmethod
     def validate_input(cls, data: Dict) -> None:
         """
         Validate input data. Must be implemented by all child "Model" classes.
         """
-        if "schema_id" not in data["parameters"] or data["parameters"]["schema_id"] == "":
+        if (
+            "schema_id" not in data["parameters"]
+            or data["parameters"]["schema_id"] == ""
+        ):
             raise PrestoBaseException("schema_id is required as input to classify", 422)
 
-        if "items" not in data["parameters"] or len(data["parameters"]["items"]) == 0:
-            raise PrestoBaseException("items are required as input to classify", 422)
+        if "input_items" not in data or len(data["input_items"]) == 0:
+            raise PrestoBaseException(
+                "input items are required as input to classify", 422
+            )
 
-        for item in data["parameters"]["items"]:
+        for item in data["input_items"]:
```

Comment:
As scott notes, this is the substantive change in this file,  now looping over the set of input items, unnested from `parameters`